packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/create-model-customization-job.ts

```diff
@@ -0,0 +1,396 @@
+                  resource: 'network-interface',
+                  resourceName: '*',
+                }),
+                stack.formatArn({
+                  service: 'ec2',
+                  resource: 'security-group',
+                  resourceName: '*',
+                }),
+                stack.formatArn({
+                  service: 'ec2',
+                  resource: 'subnet',
+                  resourceName: '*',
+                }),
+              ],
+            }),
+            new iam.PolicyStatement({
+              actions: ['ec2:CreateTags'],
+              resources: [stack.formatArn({
+                service: 'ec2',
+                resource: 'network-interface',
+                resourceName: '*',
+              })],
+              conditions: {
+                StringEquals: {
+                  'ec2:CreateAction': 'CreateNetworkInterface',
+                },
+              },
+            }),
+            new iam.PolicyStatement({
+              actions: [
+                'ec2:CreateNetworkInterfacePermission',
+                'ec2:DeleteNetworkInterface',
+                'ec2:DeleteNetworkInterfacePermission',
+              ],
+              resources: ['*'],
+              conditions: {
+                StringEquals: {
+                  'ec2:Subnet': [
+                    ...(this.props.vpcConfig
+                      ? this.props.vpcConfig.subnets.map((subnet) => subnet.subnetId)
+                      : []),
+                  ],
+                },
+              },
+            }),
+            new iam.PolicyStatement({
+              actions: ['s3:GetObject'],
+              resources: [
+                this.s3UriToArn(this.props.trainingDataS3Uri),
+                ...(this.props.validationDataS3Uri.map((s3Uri) => this.s3UriToArn(s3Uri))),
+              ],
+            }),
+            new iam.PolicyStatement({
+              actions: ['s3:PutObject'],
+              resources: [this.s3UriToArn(this.props.outputDataS3Uri)],
+            }),
+          ],
+        }),
+      },
+    });
+
+    return role;
+  }
+
+  private renderPolicyStatements(): iam.PolicyStatement[] {
+    const policyStatements = [
+      new iam.PolicyStatement({
+        actions: [
+          'bedrock:CreateModelCustomizationJob',
+          'bedrock:TagResource',
+        ],
+        resources: [
+          this.props.baseModel.modelArn,
+          Stack.of(this).formatArn({
+            service: 'bedrock',
+            resource: 'custom-model',
+            resourceName: '*',
+          }),
+          Stack.of(this).formatArn({
+            service: 'bedrock',
+            resource: 'model-customization-job',
+            resourceName: '*',
+          }),
+        ],
+      }),
+      new iam.PolicyStatement({
+        actions: ['iam:PassRole'],
+        resources: [this._role.roleArn],
+      }),
+      ...(this.props.kmsKey
+        ? [
+          new iam.PolicyStatement({
+            // TODO - this should be more specific
```

Comment:
Yes, can you share some documentation about the policy?

Reply:
I could only find this [docs](https://docs.aws.amazon.com/step-functions/latest/dg/bedrock-iam.html) which has no information about kms...

Reply:
Please check [this](https://docs.aws.amazon.com/bedrock/latest/userguide/encryption-custom-job.html).

Reply:
Thanks a lot.

I've updated the policy for the custom model builder.
```
    "Statement": [
        {
            "Sid": "Permissions for custom model builders",
            "Effect": "Allow",
            "Principal": {
                "AWS": "arn:aws:iam::account-id:user/role"
            },
            "Action": [
                "kms:Decrypt",
                "kms:GenerateDataKey",
                "kms:DescribeKey",
                "kms:CreateGrant"
            ],
            "Resource": "*"
        },
```